lumicks/pylake/tests/test_imaging_confocal/test_point_scan.py

```diff
@@ -5,19 +5,21 @@
 import matplotlib.pyplot as plt
 
 
-def test_point_scans_basic(test_point_scans, reference_timestamps, reference_counts):
-    ps = test_point_scans["PointScan1"]
-    ps_red = ps.red_photon_count
+def test_point_scans_basic(test_point_scan):
+    ps, ref = test_point_scan
 
-    assert ps_red.data.shape == (90,)
+    for color, ref_data in ref["data"].items():
+        count = getattr(ps, f"{color}_photon_count")
 
-    np.testing.assert_allclose(ps_red.timestamps, reference_timestamps)
-    np.testing.assert_allclose(ps_red.data, reference_counts)
+        assert count.data.shape == ref_data.shape
 
+        np.testing.assert_allclose(count.timestamps, ref["timestamps"])
```

Comment:
```suggestion
        np.testing.assert_equal(count.timestamps, ref["timestamps"])
```

These are integers. You should really check for equality, not closeness.

You can see that this test actually doesn't test very well right now by changing the timestamps by a small number in `conftest.py`. This will not fire.